gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=4.3.30-SNAPSHOT
+VERSION_NAME=4.3.31-SNAPSHOT
```

Comment:
Update version number to reflect added functionality

Reply:
Yeah, we should probably bump to 4.4.0